### PR TITLE
Clarify step 2 in Creating an Addon

### PR DESCRIPTION
**Pull request by DNin01**
### Changes

Added a link to the repository and its contributing guidelines from step 2 on the Creating an Addon docs page.

### Reason for changes

_Fork/clone the repo? Where do I go?_ These changes are meant to help GitHub beginners contribute.

### CODE REVIEW

**Review by WorldLanguages — APPROVED** [review]

**Comment by WorldLanguages** [comment]
We should make this page consistent with the CONTRIBUTING.md file on the ScratchAddons/ScratchAddons repo.
This is a step in that direction, but we could do it completely.

**Comment by DNin01** [comment]
> We should make this page consistent with the CONTRIBUTING.md file on the ScratchAddons/ScratchAddons repo.

Can I have more info?

**Comment by Hans5958** [comment]
Pointing to https://github.com/ScratchAddons/ScratchAddons#from-source would give three choices to clone it. Is it necessary to point to directly so they need to clone the repository?

**Comment by DNin01** [comment]
> Pointing to https://github.com/ScratchAddons/ScratchAddons#from-source would give three choices to clone it. Is it necessary to point to directly so they need to clone the repository?

Do you mean I should or I shouldn't link to the direct download?

**Review by Samq64 — COMMENTED** [review]
The current link is also fine.

File: content/docs/develop/getting-started/creating-an-addon.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ If possible, disable the Scratch Addons extension you downloaded from the store
 ## Step 1: Read about [addon basics](/docs/develop/getting-started/addon-basics/)
 Make sure to read that article to be familiar with the terminology.
 
-## Step 2: Fork/clone the repo
-Or download as ZIP, if you want. In other words, just download the source code locally.
+## Step 2: Fork/clone the [repo](https://github.com/ScratchAddons/ScratchAddons)
+Follow [these instructions](https://github.com/ScratchAddons/ScratchAddons#from-source) to download the source code locally and add Scratch Addons to your browser.
```
Comment (Samq64) [inline content/docs/develop/getting-started/creating-an-addon.md:14]:
```suggestion
Follow [these instructions](https://scratchaddons.com/docs/getting-started/installing/#from-source) to download the source code locally and add Scratch Addons to your browser.

```

**Review by Samq64 — APPROVED** [review]

**Comment by DNin01** [comment]
@WorldLanguages Ready to merge?

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**